Chaincase.Common/ChaincaseSynchronizer.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using WalletWasabi.Services;
+using WalletWasabi.Stores;
+using WalletWasabi.WebClients.Wasabi;
+
+namespace Chaincase.Common.Services
+{
+	public class ChaincaseSynchronizer : WasabiSynchronizer
+	{
+		public ChaincaseSynchronizer(Network network, BitcoinStore bitcoinStore, WasabiClient client)
+			: base(network, bitcoinStore, client)
+		{
+		}
+
+		public ChaincaseSynchronizer(Network network, BitcoinStore bitcoinStore, Func<Uri> baseUriAction, EndPoint torSocks5EndPoint)
+			: base(network, bitcoinStore, baseUriAction, torSocks5EndPoint)
+		{
+		}
+
+		public ChaincaseSynchronizer(Network network, BitcoinStore bitcoinStore, Uri baseUri, EndPoint torSocks5EndPoint)
+			: base(network, bitcoinStore, baseUri, torSocks5EndPoint)
+		{
+		}
+
+		public async Task SleepAsync()
+		{
+			Interlocked.CompareExchange(ref _running, 2, 1); // If running, make it stopping.
```

Comment:
Are these numbers (2 and 1) defined anywhere? The comment is useful

Reply:
they're defined in the WasabiSynchronizer. _running is a protected member. Maybe I should bring over exactly this as a comment too to improve legibility?

Reply:
It's also set as <summary> already so if you hover over _running in ide it should show you the definition